### PR TITLE
Cosmetic cleanup module names and WildFly stylization

### PR DESCRIPTION
**Pull request by rhusar**

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/attributes/transaction-attributes.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 :tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/TransactionManagerLookup.html[TransactionManagerLookup]
 :embedded-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/EmbeddedTransactionManagerLookup.html[EmbeddedTransactionManagerLookup]
 :jboss-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/JBossStandaloneJTAManagerLookup.html[JBossStandaloneJTAManagerLookup]
-:wildfly-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/WildflyTransactionManagerLookup.html[WildflyTransactionManagerLookup]
+:wildfly-tm-lookup: link:{javadocroot}/org/infinispan/transaction/lookup/WildflyTransactionManagerLookup.html[WildFlyTransactionManagerLookup]
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/attributes/transaction-attributes.adoc]: the class name has a lower case `f`. 
this change can be removed.

Comment (rhusar) [reply documentation/src/main/asciidoc/topics/attributes/transaction-attributes.adoc]: Oops. Let me fix that.

Comment (rhusar) [reply documentation/src/main/asciidoc/topics/attributes/transaction-attributes.adoc]: Done (though the URL would still work haha).

**Comment by pruivo** [comment]
integrated! thanks @rhusar !

**Pull request merged**